docroot/modules/custom/mymodule/mymodule.routing.yml

```diff
@@ -0,0 +1,7 @@
+mymodule.custom_settings:
+  path: '/mymodule/config'
+  defaults:
+    _title: 'mymodule custom settings'
+    _form: 'Drupal\mymodule\Form\CustomSettingsForm'
+  requirements:
+    _permission: 'administer content'
```

Comment:
we should use the permission we created as part of this module - `administer mymodule configuration`